samples/Web/OpenTelemetry.Exporter.Web.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Lets call this project AspNetCore. Why "web" ?

Reply:
Sure!

Reply:
Done!